install/disconnected_install.adoc

```diff
@@ -330,72 +313,68 @@ For example, to pull both the previous and latest version of the Tomcat image:
+$ docker pull registry.access.redhat.com/jboss-processserver-6/processserver64-openshift:<tag>
+$ docker pull registry.access.redhat.com/jboss-eap-6/eap64-openshift:<tag>
+$ docker pull registry.access.redhat.com/jboss-eap-7/eap70-openshift:<tag>
+$ docker pull registry.access.redhat.com/jboss-webserver-3/webserver31-tomcat7-openshift:<tag>
+$ docker pull registry.access.redhat.com/jboss-webserver-3/webserver31-tomcat8-openshift:<tag>
+$ docker pull registry.access.redhat.com/openshift3/jenkins-1-rhel7:<tag>
+$ docker pull registry.access.redhat.com/openshift3/jenkins-2-rhel7:<tag>
 $ docker pull registry.access.redhat.com/openshift3/jenkins-agent-maven-35-rhel7:<tag>
 $ docker pull registry.access.redhat.com/openshift3/jenkins-agent-nodejs-8-rhel7:<tag>
-$ docker pull registry.access.redhat.com/openshift3/jenkins-slave-base-rhel7
-$ docker pull registry.access.redhat.com/openshift3/jenkins-slave-maven-rhel7
-$ docker pull registry.access.redhat.com/openshift3/jenkins-slave-nodejs-rhel7
-$ docker pull registry.access.redhat.com/rhscl/mongodb-32-rhel7
-$ docker pull registry.access.redhat.com/rhscl/mysql-57-rhel7
-$ docker pull registry.access.redhat.com/rhscl/perl-524-rhel7
-$ docker pull registry.access.redhat.com/rhscl/php-56-rhel7
-$ docker pull registry.access.redhat.com/rhscl/postgresql-95-rhel7
-$ docker pull registry.access.redhat.com/rhscl/python-35-rhel7
-$ docker pull registry.access.redhat.com/redhat-sso-7/sso70-openshift
-$ docker pull registry.access.redhat.com/rhscl/ruby-24-rhel7
-$ docker pull registry.access.redhat.com/redhat-openjdk-18/openjdk18-openshift
-$ docker pull registry.access.redhat.com/redhat-sso-7/sso71-openshift
-$ docker pull registry.access.redhat.com/rhscl/nodejs-6-rhel7
-$ docker pull registry.access.redhat.com/rhscl/mariadb-101-rhel7
+$ docker pull registry.access.redhat.com/openshift3/jenkins-slave-base-rhel7:<tag>
+$ docker pull registry.access.redhat.com/openshift3/jenkins-slave-maven-rhel7:<tag>
+$ docker pull registry.access.redhat.com/openshift3/jenkins-slave-nodejs-rhel7:<tag>
+$ docker pull registry.access.redhat.com/rhscl/mongodb-32-rhel7:<tag>
+$ docker pull registry.access.redhat.com/rhscl/mysql-57-rhel7:<tag>
+$ docker pull registry.access.redhat.com/rhscl/perl-524-rhel7:<tag>
+$ docker pull registry.access.redhat.com/rhscl/php-56-rhel7:<tag>
+$ docker pull registry.access.redhat.com/rhscl/postgresql-95-rhel7:<tag>
+$ docker pull registry.access.redhat.com/rhscl/python-35-rhel7:<tag>
+$ docker pull registry.access.redhat.com/redhat-sso-7/sso70-openshift:<tag>
+$ docker pull registry.access.redhat.com/rhscl/ruby-24-rhel7:<tag>
+$ docker pull registry.access.redhat.com/redhat-openjdk-18/openjdk18-openshift:<tag>
+$ docker pull registry.access.redhat.com/redhat-sso-7/sso71-openshift:<tag>
+$ docker pull registry.access.redhat.com/rhscl/nodejs-6-rhel7:<tag>
+$ docker pull registry.access.redhat.com/rhscl/mariadb-101-rhel7:<tag>
```

Comment:
I do not think those s2i images which is not under openshift3 are following the same tag version like openshift version, e.g: the present openshfit version is 3.10.45, rhscl/nodejs-6-rhel7 has no such version tag.